### ISSUE TITLE
REST API

### ISSUE THREAD

**Issue by dmidlo** [dmidlo/histdata.com-tools#77]

**Comment by dmidlo** [comment]
## Roadmap placement after full synthetic-data audit

This issue remains deferred under #505 and depends on #526 producing a complete, deeply verified retained dataset. Serving is downstream distribution, not reconstruction evidence.

When activated:

- the API selects an immutable observed or synthetic dataset/product; it never generates, repairs, reconditions, or reinterprets ticks on request;
- dataset/product cursors bind durable identity rather than timestamp alone;
- observed and generated origins remain explicit;
- candles derive only from committed final ticks;
- unsupported account/order/trading endpoints fail explicitly;
- modern-reference, any later broker-conditioned product, and OANDA-compatible protocol fields remain separately labeled;
- the service cannot imply that HistData or synthetic products are OANDA market history.

No API framework or compatibility work should enter the v2.5 critical path. Existing contracts and public reconstruction inspection are sufficient preparation until the retained dataset exists.
